### ISSUE TITLE
ImportError: No module named packages.urllib3.exceptions

### ISSUE THREAD

**Issue by cs01** [kennethreitz/pipenv#693]
```bash
>> sudo pip install pipenv --upgrade
Requirement already up-to-date: pipenv in /usr/local/lib/python2.7/dist-packages
Requirement already up-to-date: virtualenv in /usr/local/lib/python2.7/dist-packages (from pipenv)
Requirement already up-to-date: pew>=0.1.26 in /usr/local/lib/python2.7/dist-packages (from pipenv)
Requirement already up-to-date: pip>=9.0.1 in /usr/local/lib/python2.7/dist-packages (from pipenv)
Requirement already up-to-date: pip-tools>=1.9.0 in /usr/local/lib/python2.7/dist-packages (from pipenv)
Requirement already up-to-date: requests>2.0.0 in /usr/local/lib/python2.7/dist-packages (from pipenv)
Requirement already up-to-date: flake8 in /usr/local/lib/python2.7/dist-packages (from pipenv)
Requirement already up-to-date: pathlib; python_version == "2.7" in /usr/local/lib/python2.7/dist-packages (from pew>=0.1.26->pipenv)
Requirement already up-to-date: shutilwhich; python_version == "2.7" in /usr/local/lib/python2.7/dist-packages (from pew>=0.1.26->pipenv)
Requirement already up-to-date: backports.shutil-get-terminal-size; python_version == "2.7" in /usr/local/lib/python2.7/dist-packages (from pew>=0.1.26->pipenv)
Requirement already up-to-date: pythonz-bd>=1.10.2 in /usr/local/lib/python2.7/dist-packages (from pew>=0.1.26->pipenv)
Requirement already up-to-date: virtualenv-clone>=0.2.5 in /usr/local/lib/python2.7/dist-packages (from pew>=0.1.26->pipenv)
Requirement already up-to-date: setuptools>=17.1 in /usr/local/lib/python2.7/dist-packages (from pew>=0.1.26->pipenv)
Requirement already up-to-date: click>=6 in /usr/local/lib/python2.7/dist-packages (from pip-tools>=1.9.0->pipenv)
Requirement already up-to-date: six in /usr/local/lib/python2.7/dist-packages (from pip-tools>=1.9.0->pipenv)
Requirement already up-to-date: first in /usr/local/lib/python2.7/dist-packages (from pip-tools>=1.9.0->pipenv)
Requirement already up-to-date: urllib3<1.23,>=1.21.1 in /usr/local/lib/python2.7/dist-packages (from requests>2.0.0->pipenv)
Requirement already up-to-date: idna<2.7,>=2.5 in /usr/local/lib/python2.7/dist-packages (from requests>2.0.0->pipenv)
Requirement already up-to-date: chardet<3.1.0,>=3.0.2 in /usr/local/lib/python2.7/dist-packages (from requests>2.0.0->pipenv)
Requirement already up-to-date: certifi>=2017.4.17 in /usr/local/lib/python2.7/dist-packages (from requests>2.0.0->pipenv)
Requirement already up-to-date: pyflakes<1.6.0,>=1.5.0 in /usr/local/lib/python2.7/dist-packages (from flake8->pipenv)
Requirement already up-to-date: enum34; python_version < "3.4" in /usr/local/lib/python2.7/dist-packages (from flake8->pipenv)
Requirement already up-to-date: configparser; python_version < "3.2" in /usr/local/lib/python2.7/dist-packages (from flake8->pipenv)
Requirement already up-to-date: mccabe<0.7.0,>=0.6.0 in /usr/local/lib/python2.7/dist-packages (from flake8->pipenv)
Requirement already up-to-date: pycodestyle<2.4.0,>=2.0.0 in /usr/local/lib/python2.7/dist-packages (from flake8->pipenv)
```

okay looks good, but:
```bash
>> pipenv -h
Traceback (most recent call last):
  File "/usr/local/bin/pipenv", line 9, in <module>
    load_entry_point('pipenv==7.9.5', 'console_scripts', 'pipenv')()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 570, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2751, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2405, in load
    return self.resolve()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2411, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python2.7/dist-packages/pipenv/__init__.py", line 17, in <module>
    from .cli import cli
  File "/usr/local/lib/python2.7/dist-packages/pipenv/cli.py", line 31, in <module>
    from requests.packages.urllib3.exceptions import InsecureRequestWarning
ImportError: No module named packages.urllib3.exceptions
```

some system info
```bash
>> uname -a
Linux 3.19.0-66-generic #74~14.04.1-Ubuntu SMP Tue Jul 19 19:56:11 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

>> python --version
Python 2.7.6


```

**Comment by kennethreitz** [comment]
will fix

**Comment by kennethreitz** [comment]
fixed

**Comment by cs01** [comment]
@kennethreitz thanks for the quick reqponse. 

I just upgraded to pipenv 7.9.7 but am still getting the error. Which version is it fixed in?

```bash
>> pipenv
Traceback (most recent call last):
  File "/usr/local/bin/pipenv", line 11, in <module>
    load_entry_point('pipenv==7.9.7', 'console_scripts', 'pipenv')()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 570, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2751, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2405, in load
    return self.resolve()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2411, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python2.7/dist-packages/pipenv/__init__.py", line 17, in <module>
    from .cli import cli
  File "/usr/local/lib/python2.7/dist-packages/pipenv/cli.py", line 31, in <module>
    from requests.packages.urllib3.exceptions import InsecureRequestWarning
ImportError: No module named packages.urllib3.exceptions
```

**Comment by kennethreitz** [comment]
you need to update your requests to the latest version, this will solve it.

**Comment by kennethreitz** [comment]
try latest

**Comment by kennethreitz** [comment]
7.9.10

**Comment by cs01** [comment]
Closer with 7.9.10! Thanks for helping w/ this, much appreciated. Can't wait to use pipenv.
```
>> pipenv
Traceback (most recent call last):
  File "/usr/local/bin/pipenv", line 11, in <module>
    load_entry_point('pipenv==7.9.10', 'console_scripts', 'pipenv')()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 570, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2751, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2405, in load
    return self.resolve()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2411, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python2.7/dist-packages/pipenv/__init__.py", line 17, in <module>
    from .cli import cli
  File "/usr/local/lib/python2.7/dist-packages/pipenv/cli.py", line 31, in <module>
    from urllib3.exceptions import InsecureRequestWarning
ImportError: cannot import name InsecureRequestWarning
```

pip freeze yields
```
adium-theme-ubuntu==0.3.4
apt-xapian-index==0.45
argcomplete==1.9.2
astroid==1.0.1
backports.shutil-get-terminal-size==1.0.0
boto==2.20.1
ccsm==0.9.11.3
certifi==2017.7.27.1
chardet==3.0.4
chrome-gnome-shell==0.0.0
click==6.7
colorama==0.2.5
command-not-found==0.3
compizconfig-python==0.9.11.3
configparser==3.5.0
crcmod==1.7
debtagshw==0.1
decorator==3.4.0
defer==1.0.6
dirspec==13.10
distro-info==0.12
docutils==0.11
duplicity==0.6.23
enum34==1.1.6
first==2.0.1
flake8==3.4.1
funcsigs==1.0.2
gcs-oauth2-boto-plugin==1.14
google-apitools==0.5.3
gsutil==4.27
html5lib==0.999
httplib2==0.8
idna==2.6
ipython==1.2.1
Jinja2==2.7.2
lockfile==0.8
logilab-common==0.61.0
lxml==3.3.3
MarkupSafe==0.18
matplotlib==1.3.1
mccabe==0.6.1
mercurial==2.8.2
mock==2.0.0
numpy==1.8.2
oauth2client==2.2.0
oauthlib==0.6.1
oneconf==0.3.7.14.4.1
PAM==0.4.2
paramiko==1.10.1
pathlib==1.0.1
pbr==3.1.1
pew==0.1.26
pexpect==3.1
Pillow==2.3.0
pip-tools==1.9.0
pipenv==7.9.10
piston-mini-client==0.7.5
psutil==1.2.1
pyasn1==0.3.5
pyasn1-modules==0.1.4
pycodestyle==2.3.1
pycrypto==2.6.1
pycups==1.9.66
pyflakes==1.5.0
Pygments==1.6
pygobject==3.12.0
pylint==1.1.0
pyOpenSSL==0.13
pyparsing==2.0.1
pyserial==2.6
pysmbc==1.0.14.1
python-apt===0.9.3.5ubuntu2
python-cloudfiles==1.7.10
python-dateutil==1.5
python-debian===0.1.21-nmu2ubuntu2
python-gflags==3.1.1
pythonz-bd==1.11.4
pytz==2012rc0
pyxdg==0.25
PyYAML==3.10
pyzmq==14.0.1
quantities==0.10.1
reportlab==3.0
requestbuilder==0.1.0b1
requests==2.18.4
retry-decorator==1.1.0
roman==2.0.0
rope==0.9.2
rsa==3.4.2
scipy==0.13.3
semver==2.7.8
shutilwhich==1.1.0
simplegeneric==0.8.1
six==1.11.0
SocksiPy-branch==1.1
software-center-aptd-plugins==0.0.0
Sphinx==1.2.2
spyder==2.2.5
ssh-import-id==3.21
stevedore==1.26.0
system-service==0.1.6
unity-lens-photos==1.0
urllib3==1.22
virtualenv==15.1.0
virtualenv-clone==0.2.6
virtualenvwrapper==4.8.1
xdiagnose===3.6.3build2
```

**Comment by kennethreitz** [comment]
I highly recommend the fancy installation method

https://docs.pipenv.org/en/latest/advanced.html#fancy-installation-of-pipenv

e.g. you have a lot of conflicting shit installed :)

**Comment by kennethreitz** [comment]
`from urllib3.exceptions import InsecureRequestWarning` should absolutely work with urllib3 v1.22

**Comment by cs01** [comment]
Ugh, I had the apt get package `python-urllib3` installed on my system, so python was finding the `dist-package` and using version `1.7.1` rather than the pip-installed `1.22`. (not sure how the dist package version was higher?). I uninstalled with `sudo apt-get remove python-urllib3`, and `pipenv` works now. Thanks for the help.

On a related note, I found the installation instructions a little confusing. Before you recommended the "Fancy Installation", I had already seen that page and that section. But when I read it, it was unclear why someone would want to use "Fancy" vs "Pragmatic" vs "Crude". May I suggest updating the sections for who would want to use it and why, and possibly renaming "Fancy" to something more descriptive?
